src/libusb/enumerator-libusb.cpp

```diff
@@ -0,0 +1,83 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#include "usb/usb-enumerator.h"
+#include "libusb/device-libusb.h"
+#include "types.h"
+
+#include <libusb.h>
+
+namespace librealsense
+{
+    namespace platform
+    {
+        struct usb_context
+        {
+            usb_context() { libusb_init(&_ctx); }
+            ~usb_context() { libusb_exit(_ctx); }
+            libusb_context* get() { return _ctx; }
+        private:
+            struct libusb_context* _ctx;
+        };
+
+        std::shared_ptr<usb_context> get_context()
+        {
+            static std::shared_ptr<usb_context> context = std::make_shared<usb_context>();
+            return context;
+        }
+
+        bool usb_enumerator::is_device_connected(const std::shared_ptr<usb_device>& device)
+        {
+            if (device == nullptr)
+                return false;
+            std::vector<std::shared_ptr<usb_device>> rv;
+            ssize_t count = 0;
+            libusb_device **list = NULL;
+
+            auto ctx = get_context()->get();
+            count = libusb_get_device_list(ctx, &list);
+
+            bool found = false;
+            for (ssize_t idx = 0; idx < count; ++idx)
+            {
+                libusb_device *lusb_device = list[idx];
+                libusb_device_descriptor desc = { 0 };
+
+                auto rc = libusb_get_device_descriptor(lusb_device, &desc);
+                if (desc.idVendor != 0x8086)
```

Comment:
For review - probably requires some sort of a mask - T265 , etc'
Log the devices that are filtered out

Reply:
device are filtered on context.cpp via pick_<X>_devices